scheduler/scheduler.go

```diff
@@ -156,7 +156,16 @@ func (s *Scheduler) Schedule(ctx context.Context) error {
 					"expected_duration_span": res.schedulerEvent.ExpectedDuration.String(),
 					"span":     time.Since(distroStartTime).String(),
 					"duration": time.Since(distroStartTime),
-				})
+				}
+				latencies, err := model.AverageTaskLatencyByDistro(d.distroId, time.Minute)
+				if err != nil {
+					grip.Error(errors.Wrap(err, "error getting latencies"))
+				} else {
+					for r, d := range latencies {
+						m[r+"_duration"] = d
```

Comment:
This is duration in nanoseconds right? Can we grip log something more human readable, like the .String() form of the duration?